### PR TITLE
Fix typo in index.md

### PR DESCRIPTION
**Pull request by badersur**

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/standard/index?branch=pr-en-us-1037](https://review.docs.microsoft.com/en-us/dotnet/articles/standard/index?branch=pr-en-us-1037)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C19%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609191829026730-1037%5Cworkflow_report.html).

File: docs/standard/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
 
 > Check out the ["Getting Started with .NET Core" tutorials](../core/getting-started.md) to learn how to create a simple .NET Core application. It only takes a few minutes to get your first app up and running.
 
-.NET is a general purpose development platform. It can be used for any kind of app type or workload where general purpose solutions are used. It has several key features that are attractive to many developers, including automatic memory management and modern programming languages, that make it easier to efficiently build high-quality applications. .NET enables a high-level programming environment with many convenience features, while providing low-level access to native memory and APIs.
+.NET is a general purpose development platform. It can be used for any kind of app type or workload where general purpose solutions are used. It has several key features that are attractive to many developers, including automatic memory management and modern programming languages, that make it easier to efficiently build high-quality applications. .NET enables a high-level programming environment with many convenient features, while providing low-level access to native memory and APIs.
```
Comment (svick) [inline docs/standard/index.md]: I think this is not a typo, it's jargon. As I understand it:

* a convenient feature: feature that is convenient to use
* a convenience feature: feature whole sole purpose is to make some task more convenient

Comment (stevehoag) [inline docs/standard/index.md]: @richlander Which was your intended meaning?

Comment (badersur) [inline docs/standard/index.md]: It could be but it seems weird to me! My mind couldn't accept it...

Comment (mairaw) [inline docs/standard/index.md]: I also think that was the original intention

Comment (badersur) [inline docs/standard/index.md]: It seems I was wrong and maybe I should close this PR!

Comment (mairaw) [inline docs/standard/index.md]: Let's close it then! But don't hesitate to ask or suggest improvements. Thanks!

Comment (badersur) [inline docs/standard/index.md]: Welcome. I won't stop suggesting improvements until I revise C# and understand .NET Core..

**Pull request closed without merging**